lib/engine.js

```diff
@@ -21,13 +21,16 @@ module.exports = function (s, conf) {
 
   let so = s.options
   if(_.isUndefined(s.exchange)){
-    if (so.mode == 'sim') {
+    if (so.mode !== 'live') {
```

Comment:
this line should also catch line 31. or just to so.mode == 'sim' || so.mode == 'paper'   (so everyone knows whats going on)

Reply:
It doesn't. In paper mode, `s.exchange` is already defined so the check on line 23 is false.

Reply:
in paper so.mode isnt paper? (which is also so.mode !== 'live')

Reply:
ok, then i dont understand when line 27 is reached. but that's none of your business.

Reply:
Maybe in `mode === 'live'` or `mode === 'train'` which is another mode I've never used but exists.

Reply:
yeah but why is the exchange defined in paper in line 23, but not in live when reaching the line? Whats what i meant. But as i said, no need for explanation,

Reply:
I think it actually is defined there in live as well because `trade.js` defines it, and `trade.js` is what's running for both live and paper. So maybe my check is only REALLY catching against train mode? Which, again, I'm not sure in that case that I'm doing this right.

Reply:
This change could be the reason to the #1410 issue.